pyopensprinkler/__init__.py

```diff
@@ -1,84 +1,165 @@
 """Main OpenSprinkler module."""
 
+import hashlib
 import json
 import urllib
 
 import httplib2
 from backoff import expo, on_exception
 
-from pyopensprinkler.device import Device
 from pyopensprinkler.program import Program
 from pyopensprinkler.station import Station
 
-_HTTP = httplib2.Http()
 
+class Controller(object):
```

Comment:
`Controller` seems a bit too generic as the main class name, maybe keep `OpenSprinkler` but leave the controller references within the class itself?

Reply:
The library itself is the namespacing right? If someone wants to be more specific they can `import as` if I understand correctly.

Reply:
Ok